xline/src/server/command.rs

```diff
@@ -305,14 +305,10 @@ where
     fn prepare(
         &self,
         cmd: &Command,
-        index: LogIndex,
+        _index: LogIndex,
```

Comment:
Please remove the index if it not needed

Reply:
Removed